packages/nx/src/command-line/exec/exec.ts

```diff
@@ -21,9 +26,20 @@ import { workspaceRoot } from '../../utils/workspace-root';
 import { calculateDefaultProjectName } from '../../config/calculate-default-project-name';
 
 export async function nxExecCommand(
-  args: Record<string, string[]>
+  args: Record<string, any>
```

Comment:
Why is this not typed? If string was too strict surely we can expand it.

Reply:
done. listed out all the possible values instead.